code/modules/mob/mob.dm

```diff
@@ -1839,11 +1839,9 @@ Use this proc preferably at the end of an equipment loadout
 	return FALSE
 
 /mob/proc/isTeleViewing(var/client_eye)
-	if(istype(client_eye,/obj/machinery/camera))
-		return 1
-	if(istype(client_eye,/obj/item/projectile/rocket/nikita))
-		return 1
-	return 0
+	return istype(client_eye,/obj/machinery/camera) ||\
+			istype(client_eye,/obj/item/projectile/rocket/nikita) ||\
```

Comment:
The original style is considerably more readable than this version, and I don't think there's a considerable speed difference.

Reply:
honestly i was thinking of even changing it to a define too, but if you think it'd be better as is, i'll keep it like that

Reply:
I'd lean to the more readable version myself. I don't think this really fits as a define since it's a little bit more complicated than a simple list of isTypes with the new adjacency check.

Reply:
sounds fine to me, fixed